### DISCUSSION TITLE
[V3] Alpine.JS template breaks when using wire:navigate

### DISCUSSION THREAD

**Discussion by cwkhawand** [livewire/livewire#7151] [7. Bugs (archived)]
### Livewire version

3.0.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create 2 pages that use wire:navigate to navigate between one another
2. Create a template on page 1 that uses x-if to conditionally show a div (ideally x-if should always evaluate to true)
3. Navigate to page 2 using wire:navigate
4. Click the back button of the browser
5. The conditionally shown div will appear twice (once because of the old HTML, and a second time because of the new template render)
6. Repeat as many times as you want (click link + navigate back), the template will appear as many times as you navigated

I'm assuming this is due to the fact that:
1. Alpine.JS creates an HTML element to show the template
2. When navigating, Livewire saves the HTML with the new element
3. When going back, the template is re-executed but the old element is not cleared so it appears twice, thrice, etc...

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by cwkhawand** [comment]
Hey! Any updates?

**Comment by danie-ramdhani** [comment]
you can try this workaround in this discussion https://github.com/livewire/livewire/discussions/6849

or wait bug fix in https://github.com/livewire/livewire/pull/7034
